mrp/src/mrp/cmd/up.py

```diff
@@ -79,13 +79,13 @@ def callback(system_state):
                 ns.sat = True
                 ns.cv.notify()
 
-    watcher = life_cycle.system_state_watcher(callback)
+    life_cycle.system_state_watcher(callback)
```

Comment:
The `watcher` variable is required. It manages the lifetime of the thread that executes the callback.

Reply:
Thanks for catching it. However `flake8` views `watcher` as an unused variable and complains about it in the test. What do you think is the best way to handle this?

Reply:
Maybe I can add an ignore list to ignore these type of rules?

Reply:
Can we just annotate that one line:
```py
watcher = life_cycle.system_state_watcher(callback)  # noqa: F841
```

Reply:
My concern is that all future uses of watcher will then need to have that annotation, not sure if this is scalable.